ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
-
-**Example:**
-
--	**`apt-get install debhelper`**
--	**`apt-get install dh-autoreconf`**
--	**`etc. etc. etc.`** 
+git clone https://github.com/ArcticaProject/remote-logon-service/
+```
+If done correctly then the Terminal will tell you, **"Checking connectivity… done"**.
 
-The following packages that need to be installed:
 
--	**`debhelper`**
--	**`dh-autoreconf`**
--	**`fonts-droid-fallback`**
--	**`gnome-common`**
--	**`gnome-settings-daemon-dev`**
+## Stage Four 
 
-In order to install libayatana you will need to update the packages type following:
+Comment from sunweaver: CONTENT adaptation!!! Just provide one long apt-get line per source package that pulls in all required packages. IIRC, there also is a tool that reads build-deps from a debian/control file. This section can be improved!
 
-- **`apt-get update`**
 
-Now carry on with **"`apt-get install`"** followed by the command.
+Install the dependencies for the four packages which you acquired from Git. For this you will need to run the command `apt-get install` on the following packages. When installing it will sometimes ask you if you want to continue type **"y"** and press enter.
 
--	**`libayatana-ido3-0.4-dev`**
--	**`libayatana-indicator3-dev`**
--	**`libcanberra-dev`**
--	**`libgnome-desktop-3-dev`**
--	**`libgtk-3-dev`**
--	**`liblightdm-gobject-1-dev`**
--	**`valac`**
--	**`xvfb`**
--	**`fonts-cantarell`**
--	**`fonts-cabin`**
--	**`gnome-settings-daemon`**
+**Example:**
 
-Install libpam-x2go dependencies, use the same command **"`apt-get install`"** followed by the dependencies:
+```
+# apt-get install debhelper
+# apt-get install dh-autoreconf
+# etc. etc. etc. 
+```
 
--	**`libgtest-dev`**
--	**`libpam0g-dev`**
--	**`libssh-dev`** 
--	**cdbs** 
--	**`socat`**
--	**`pyhoca-cli`**
+The following packages that need to be installed:
+```
+# apt-get install debhelper
+# apt-get install dh-autoreconf
+# apt-get install fonts-droid-fallback
+# apt-get install gnome-common
+# apt-get install gnome-settings-daemon-dev
+```
 
+In order to install libayatana you will need to update the packages type following:
+```
+# apt-get update
+```
+
+Now carry on with `apt-get install` followed by the command.
+
+```
+# apt-get install libayatana-ido3-0.4-dev
+# apt-get install libayatana-indicator3-dev
+# apt-get install libcanberra-dev
+# apt-get install libgnome-desktop-3-dev
+# apt-get install libgtk-3-dev
+# apt-get install liblightdm-gobject-1-dev
+# apt-get install valac
+# apt-get install xvfb
+# apt-get install fonts-cantarell
+# apt-get install fonts-cabin
+# apt-get install gnome-settings-daemon
+```
+
+Install libpam-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
+
+```
+# apt-get install libgtest-dev
+# apt-get install libpam0g-dev
+# apt-get install libssh-dev 
+# apt-get install cdbs 
+# apt-get install socat
+# apt-get install pyhoca-cli
```

Comment:
make sure, you _only_ list packages from Build-Depends: field in debian/control of the libpam-x2go package. Please check all other dependencies, too.